build.gradle

```diff
@@ -44,7 +44,8 @@ run {
 }
 
 repositories {
-    mavenCentral()
+        mavenCentral()
+        maven { url "http://central.maven.org/maven2" }
```

Comment:
Yes, I think so. In my case (while doing 'gradle distZip') I got an error 'Could not HEAD 'https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.3.0/jackson-databind-2.3.0.pom' when just mavenCentral() was specified. In this case gradle seems to download artifacts (no repo before/ just after gradle installation) from repo1.maven.org/maven2 and it failed. Adding the central cerver explicitly helped.